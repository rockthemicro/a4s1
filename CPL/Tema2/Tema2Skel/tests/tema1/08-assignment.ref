"08-assignment.cl", line 8:5, Semantic error: Class A redefines method f
Semantic error: No method main in class Main
Compilation halted
