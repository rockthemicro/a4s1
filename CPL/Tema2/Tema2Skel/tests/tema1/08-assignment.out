line 1:0 token recognition error at: 'c'
line 1:1 token recognition error at: 'l'
line 1:2 token recognition error at: 'a'
line 1:3 token recognition error at: 's'
line 1:4 token recognition error at: 's'
line 1:6 token recognition error at: 'A'
line 1:8 token recognition error at: '{'
line 2:4 token recognition error at: 'x'
line 2:6 token recognition error at: ':'
line 2:8 token recognition error at: 'I'
line 2:9 token recognition error at: 'n'
line 2:10 token recognition error at: 't'
line 2:11 token recognition error at: ';'
line 4:4 token recognition error at: 'f'
line 4:5 token recognition error at: '('
line 4:6 token recognition error at: 'y'
line 4:8 token recognition error at: ':'
line 4:10 token recognition error at: 'I'
line 4:11 token recognition error at: 'n'
line 4:12 token recognition error at: 't'
line 4:13 token recognition error at: ')'
line 4:15 token recognition error at: ':'
line 4:17 token recognition error at: 'I'
line 4:18 token recognition error at: 'n'
line 4:19 token recognition error at: 't'
line 4:21 token recognition error at: '{'
line 5:8 token recognition error at: 'x'
line 5:10 token recognition error at: '<'
line 5:11 token recognition error at: '-'
line 5:13 token recognition error at: 'y'
line 6:4 token recognition error at: '}'
line 6:5 token recognition error at: ';'
line 8:4 token recognition error at: 'f'
line 8:5 token recognition error at: '('
line 8:6 token recognition error at: 'y'
line 8:8 token recognition error at: ':'
line 8:10 token recognition error at: 'I'
line 8:11 token recognition error at: 'n'
line 8:12 token recognition error at: 't'
line 8:13 token recognition error at: ','
line 8:15 token recognition error at: 'z'
line 8:17 token recognition error at: ':'
line 8:19 token recognition error at: 'I'
line 8:20 token recognition error at: 'n'
line 8:21 token recognition error at: 't'
line 8:22 token recognition error at: ')'
line 8:24 token recognition error at: ':'
line 8:26 token recognition error at: 'I'
line 8:27 token recognition error at: 'n'
line 8:28 token recognition error at: 't'
line 8:30 token recognition error at: '{'
line 9:8 token recognition error at: 'x'
line 9:10 token recognition error at: '<'
line 9:11 token recognition error at: '-'
line 9:13 token recognition error at: 'y'
line 9:15 token recognition error at: '<'
line 9:16 token recognition error at: '-'
line 9:18 token recognition error at: 'z'
line 10:4 token recognition error at: '}'
line 10:5 token recognition error at: ';'
line 11:0 token recognition error at: '}'
line 11:1 token recognition error at: ';'
